project_detail_report/views/project_report.xml

```diff
@@ -0,0 +1,130 @@
+                                                    <t t-esc="'%02d:%02d' % (int(t.planned_hours), t.planned_hours % 1 * 60)"/></t>
+                                                <t t-if="not t.planned_hours">00:00</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.effective_hours'>
+                                                    <t t-esc="'%02d:%02d' % (int(t.effective_hours), t.effective_hours % 1 * 60)"/></t>
+                                                <t t-if="not t.effective_hours">00:00</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.remaining_hours'>
+                                                    <t t-esc="'%02d:%02d' % (int(t.remaining_hours), t.remaining_hours % 1 * 60)"/></t>
+                                                <t t-if="not t.remaining_hours">00:00</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.user_id'>
+                                                    <span t-field="t.user_id"/>
+                                                </t>
+                                                <t t-if="not t.user_id">-</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.date_assign'>
+                                                    <span t-field="t.date_assign"/>
+                                                </t>
+                                                <t t-if="not t.date_assign">-</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.date_deadline'>
+                                                    <span t-field="t.date_deadline"/>
+                                                </t>
+                                                <t t-if="not t.date_deadline">-</t>
+                                            </td>
+                                            <td>
+                                                <t t-if='t.stage_id'>
+                                                    <span t-field="t.stage_id"/>
+                                                </t>
+                                                <t t-if="not t.stage_id">-</t>
+                                            </td>
+                                        </tr>
+                                    </t>
+                                </tbody>
+                            </table>
+                        </t>
+                    </div>
+                </form>
+            </div>
+        </t>
+    </t>
+    </t>
+</template>
+</odoo>
```

Comment:
review done k baki ?

Reply:
Done and also modules new db ma installed kari ne functional pan check karya chhe.